src/node-api-dotnet/init.js

```diff
@@ -42,10 +42,28 @@ function initialize(targetFramework) {
   }
 
   const assemblyName = 'Microsoft.JavaScript.NodeApi';
-  const nativeHostPath = __dirname + `/${rid}/${assemblyName}.node`;
-  const managedHostPath = __dirname + `/${targetFramework}/${assemblyName}.DotNetHost.dll`
+  let nativeHost;
+  switch(rid) {
```

Comment:
@sbowler  thank you for the fix! Could you add a comment in the code explaining why the switch is needed. This is to prevent in future someone wanting to optimize it.

Reply:
I've added some documentation that I think should help clarify the need and impact to future maintainers.